types/testing-library__jest-dom/test/testing-library__jest-dom-extend-matchers-tests.ts

```diff
@@ -70,54 +70,5 @@ customExpect(element).toHaveErrorMessage('Invalid time: the time must be between
 customExpect(element).toHaveErrorMessage(/invalid time/i);
 customExpect(element).toHaveErrorMessage(expect.stringContaining('Invalid time'));
 
-customExpect(element).not.toBeInTheDOM();
```

Comment:
@kryops How come you removed all the `.not` test cases?

Reply:
The testing library types actually do not contain `not`, only the Jest types do.

These tests were only green because the types mapped every unkown property to `any` before, but I don't think they really made sense.

Reply:
Ah I see, because of the `customExpect`. Looks like the jest suite is still testing `.not` so this looks good :)